testing/pac-ts-eks/tests/index.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright 2016-2020, Pulumi Corporation.  All rights reserved.
+
+import * as aws from "@pulumi/aws";
+import * as policy from "@pulumi/policy";
+import * as pulumi from "@pulumi/pulumi";
+
+const stackPolicy: policy.StackValidationPolicy = {
+    name: "eks-test",
+    description: "EKS integration tests.",
+    enforcementLevel: "mandatory",
+    validateStack: async (args, reportViolation) => {
+        const clusterResource = args.resources.find(r => r.isType(aws.eks.Cluster));
+        const cluster = clusterResource && clusterResource.asType(aws.eks.Cluster);
+        if (!cluster) {
+            reportViolation("EKS Cluster not found");
+            return;
+        }
+
+        if (cluster.version !== "1.13") {
+            reportViolation(`Expected EKS Cluster '${cluster.name}' version to be '1.13' but found '${cluster.version}'`);
+        }
+
+        const vpcId = cluster.vpcConfig.vpcId;
+        if (!vpcId) {
+            if (!pulumi.runtime.isDryRun()) {
+                reportViolation(`EKS Cluster '${cluster.name}' has unknown VPC`);
+            }
+            return;
+        }
+
+        const ec2 = new aws.sdk.EC2({region: aws.config.region});
```

Comment:
I haven't tried this, but can't we use `.getProvider()` and then get the region from there? This way, we use the resource's region rather than rely on `aws.config.region` which may not necessarily be the same? It happens to be so in this case but that assumes some level of whitebox knowledge.

Reply:
So the `cluster` thing is not a proper resource but rather a property bag. An advantage of that is that there are no outputs to work around. But likewise, there's not `.getProvider()` to call directly. @justinvp do you know if there's an easy way to get the region of a given resource in PaC?